homeassistant/components/discord/notify.py

```diff
@@ -129,10 +129,10 @@ async def async_send_message(self, message: str, **kwargs: Any) -> None:
         embeds: list[nextcord.Embed] = []
         if ATTR_EMBED in data:
             embedding = data[ATTR_EMBED]
-            title = embedding.get(ATTR_EMBED_TITLE) or nextcord.Embed.Empty
-            description = embedding.get(ATTR_EMBED_DESCRIPTION) or nextcord.Embed.Empty
-            color = embedding.get(ATTR_EMBED_COLOR) or nextcord.Embed.Empty
-            url = embedding.get(ATTR_EMBED_URL) or nextcord.Embed.Empty
+            title = embedding.get(ATTR_EMBED_TITLE, None)
+            description = embedding.get(ATTR_EMBED_DESCRIPTION, None)
+            color = embedding.get(ATTR_EMBED_COLOR, None)
+            url = embedding.get(ATTR_EMBED_URL, None)
```

Comment:
```suggestion
            title = embedding.get(ATTR_EMBED_TITLE)
            description = embedding.get(ATTR_EMBED_DESCRIPTION)
            color = embedding.get(ATTR_EMBED_COLOR)
            url = embedding.get(ATTR_EMBED_URL)
```

None is already the default value.